test/common/config/grpc_mux_impl_test.cc

```diff
@@ -311,6 +305,31 @@ TEST_F(GrpcMuxImplTest, WatchDemux) {
   expectSendMessage(type_url, {}, "2");
 }
 
+// Validate behavior when Cluster sends empty updates.
```

Comment:
I would phrase it a bit differently; we want one test when there is a single watcher with empty updates and another test when we have multiple watchers with an empty update.